.circleci/config.yml

```
@@ -27,6 +27,24 @@ commands:
           command: git submodule update --init --recursive <<parameters.paths>>
           environment:
             GIT_LFS_SKIP_SMUDGE: 1
+  build-headless:
+    steps:
+      - run: git -C NineChronicles.Headless log --pretty="%T" -n 1 > tree-hash
```

```suggestion
      - run: git rev-parse HEAD:NineChronicles.Headless > tree-hash
```

rev-parse can give you the tree-ish hash in simpler terms: https://git-scm.com/docs/git-rev-parse#Documentation/git-rev-parse.txt-emltrevgtltpathgtemegemHEADREADMEememmasterREADMEem